x-pack/plugins/cases/server/routes/api/cases/status/get_status.test.ts

```diff
@@ -47,17 +46,17 @@ describe('GET status', () => {
     const response = await routeHandler(context, request, kibanaResponseFactory);
     expect(context.core.savedObjects.client.find).toHaveBeenNthCalledWith(1, {
       ...findArgs,
-      filter: `((cases.attributes.status: open AND cases.attributes.type: individual) OR cases.attributes.type: ${CaseType.collection})`,
+      filter: expect.anything(),
```

Comment:
I couldn't figure out how to get the `Kquery` objects to line up with the unit tests 🤷‍♂️

Reply:
I think you can use `toElasticsearchQuery` or `toQueryStringQuery` to test it.

Reply:
Oh good point, I'll give that a shot